test/blocks/ost/ost.test.js

```diff
@@ -41,6 +58,9 @@ describe('test createLinkMarkup', () => {
   });
 
   it('create a "price" link', async () => {
+    const EXPECTED_PRICE_TEXT = `{{PRICE - ${offerType} - Stock}}`;
```

Comment:
why do we keep `{{` around the price text?

Reply:
> why do we keep `{{` around the price text?

There's a separate story to address pricing links from OST.